package.xml

```diff
@@ -13,11 +18,21 @@
   <build_depend>ed</build_depend>
   <build_depend>geometry_msgs</build_depend>
   <build_depend>sensor_msgs</build_depend>
-  <build_depend>tf</build_depend>
+  <build_depend>tf2</build_depend>
+  <build_depend>tf2_geometry_msgs</build_depend>
```

Comment:
Depend?

Reply:
`depend` is actually the lazy way.
`depend` means `build_depend`, `exec_depend` and `build_export_depend`. But none of these packages are actually included in exposed header files, as there aren't any. So therefore `depend` is not the most correct way to go.